lib/snowplow.dart

```diff
@@ -32,16 +32,20 @@ class Snowplow {
   ///
   /// [endpoint] refers to the Snowplow collector endpoint.
   /// [method] is the HTTP method used to send events to collector and it defaults to POST.
+  /// [customPostPath] is an optional string for custom POST collector paths. Do not include a starting "/".
```

Comment:
Nit: Do you think we could be a bit more smart here and remove the leading slash if users provide it? It might make it a bit less frustrating for users who don't read the API docs.